WordPress/Classes/Models/AbstractPost.m

```diff
@@ -477,7 +477,7 @@ - (BOOL)supportsStats
 
 - (BOOL)isPrivate
 {
-    return self.blog.isPrivate;
+    return self.blog.isPrivateAtWPCom;
```

Comment:
When this was a pass-thru call to the blog it was clear just by the method names what was intended. Now there is a bit of a naming mismatch and I'm wondering about future confusion.  Maybe add a comment to clarify things?

Reply:
So many thoughts around this.  I'll share them with you so you can evaluate what I did here - please bear with me for a second since this isn't straightforward.

### Why I went for a rename:

At first I was using `blog.isPrivate`.   But I soon found out that it wasn't working at all for Jetpack sites - hence Atomic Private sites.

The reason is that (the original) `blog.isPrivate` was also checking that the blog was hosted at WPcom.

This was a code smell IMHO since the name `isPrivate` doesn't convey that additional check.

### The two alternatives:

So I needed to either change the logic in that method for the whole App (risking breakage), or renaming it, and creating a cleaner `isPrivate` method that only checks if a blog is private, regardless of where it's hosted.

I opted for the second solution, as it would ensure the rest of the App would still keep working as it has been so far, and I would still get a new cleaner method for Atomic Private  sites.

### A possible third alternative:

As I mentioned above, my hunch is that we probably shouldn't be checking if a blog is hosted in WPcom, but  rather if it's accessible through WPcom.

I didn't want to dive too deep into this though, as it's easy to get lost in the rabbit hole.

### Going back to your review!

So that's the background.

I agree with what you're saying here.  Maybe what I can do is rename this method too so that it's also `isPrivateAtWPCom`, and then create a cleaner `isPrivate` method that redirects to `blog.isPrivate`.

Thoughts?

Reply:
Thanks for the background! 
That works for me. It would avoid any confusion that way :)  I had started to suggest renaming but I was worried there would be a massive number of places that would need to be updated. 😬  
If it's a chore I think a comment would probably suffice. Up to you!

Reply:
Done!